api/v1alpha3/cluster_types.go

```diff
@@ -36,6 +38,9 @@ type ClusterSpec struct {
 	// +optional
 	Paused bool `json:"paused,omitempty"`
 
+	// PostApplyAddons is a list of Secrets in YAML format to be applied to remote clusters.
```

Comment:
The comment should give enough information on how to structure the secrets, when a user can expect these yaml to be installed.

I'd also remove `remote` part in `remote clusters`, given that the field is on a Cluster object